routes/myRoute.js

```diff
@@ -0,0 +1,27 @@
+var express =require('express')
+var app=express();
+var myrouter =express.Router();
+myrouter.use(express.json())
+
+myrouter.post("/",(req,res)=>
+{
+   
+var i_data=(req.body);
+
+var original_output=JSON.stringify(i_data.payload);
+
+Object.keys(i_data.referenceData).
+forEach(key =>
+{
+   var change =`{${key}}`;
+
+   var output = new RegExp(change,"g");
+
+   original_output = original_output.replace(output,i_data.referenceData[key])
```

Comment:
can you test with bigger payload? say 4MB, 10MB etc? and see if the code works without the any system hang?